doc/whatsnew/2.6.rst

```diff
@@ -14,6 +14,13 @@ Summary -- Release highlights
 New checkers
 ============
 
+* Add `nan-comparison` check for comparison of NaN values
+
+* Add support to ``ignored-argument-names`` in DocstringParameterChecker and
+  adds `useless-param-doc` and `useless-type-doc` messages.
+
+* Add `empty-comment` check for empty comments.
+
```

Comment:
Thank you for merging master in this. I think you still need to move this text from 2.6.rst to 2.7.rst :)